sandbox/lib/presets.dart

```diff
@@ -5,8 +5,8 @@ import 'entities/max_acceleration_entity.dart';
 import 'entities/static_target_entity.dart';
 import 'scene.dart';
 
-final kPresets = <Scene>[
-  _seek(),
+final kPresets = [
```

Comment:
I don't know if you saw the comment on the merged PR, but we are following this standard, not the internal dart standard.
 https://dart.dev/guides/language/effective-dart/style#dont-use-prefix-letters

Reply:
i'll be replacing this with a class in the next PR anyways